docs/source/reference/frame.rst

```diff
@@ -65,6 +65,7 @@ Computations / Descriptive Stats
    :toctree: api/
 
    DataFrame.abs
+   DataFrame.clip
```

Comment:
cc @floscha I realized we forgot to update the api doc page with your clip change

Reply:
Thanks for adding them here